boms/cloud-oss-bom/pom.xml

```diff
@@ -66,6 +67,58 @@
         <version>${protobuf.version}</version>
       </dependency>
 
+      <!-- google-http-java-client from https://github.com/googleapis/google-http-java-client -->
+     <dependency>
+       <groupId>com.google.http-client</groupId>
+       <artifactId>google-http-client</artifactId>
```

Comment:
Why not import google-http-client-bom instead of listing these dependencies?

Reply:
That's really unfortunate. Have you filed an issue?

Reply:
I fixed two of them, and that should be out in 1.30.0. The icu4j issue I don't full understand yet, though fixing jaxen would probably render it moot.

Reply:
Could you add a TODO to migrate to google-http-client-bom? (or file an issue in cloud-opensource-java)

Reply:
https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/541